src/fmpz_mod_mpoly_q/print_pretty.c

```diff
@@ -0,0 +1,66 @@
+/*
+    Copyright (C) 2020 Fredrik Johansson
+
+    This file is part of FLINT.
+
+    FLINT is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.  See <https://www.gnu.org/licenses/>.
+*/
+
+#include "fmpz_mod_mpoly_q.h"
+#include "gr.h"
+
+void
+fmpz_mod_mpoly_q_print_pretty(const fmpz_mod_mpoly_q_t f, const char ** x, const fmpz_mod_mpoly_ctx_t ctx)
+{
+    if (fmpz_mod_mpoly_is_one(fmpz_mod_mpoly_q_denref(f), ctx))
+    {
+        fmpz_mod_mpoly_print_pretty(fmpz_mod_mpoly_q_numref(f), x, ctx);
+    }
+    else if (fmpz_mod_mpoly_is_fmpz(fmpz_mod_mpoly_q_denref(f), ctx))
```

Comment:
I don't think this branch can be reached. If the denominator is a constant, it should be 1.